src/servers/Http1Server.cc

```diff
@@ -394,12 +392,14 @@ Http::One::Server::noteTakeServerConnectionControl(ServerConnectionContext serve
 ConnStateData *
 Http::NewServer(const MasterXaction::Pointer &xact)
 {
+    debugs(11, DBG_PROTOCOL, "switch to HTTP on: " << xact->tcpClient);
     return new Http1::Server(xact, false);
 }
 
 ConnStateData *
 Https::NewServer(const MasterXaction::Pointer &xact)
 {
+    debugs(11, DBG_PROTOCOL, "switch to HTTP on: " << xact->tcpClient);
```

Comment:
Please remove this line. This change is highly controversial (e.g., I do not think we are switching to HTTP here) and lies outside the declared PR scope.

Reply:
An HTTP server AsyncJob is being created and assigned ownership of the connection (per your definition of Comm::Connection lifecycle vs ownership discussions). This should not be controversial.